WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -988,11 +1002,11 @@ class AbstractPostListViewController : UIViewController, WPContentSyncHelperDele
         }
     }
 
-    func keyForCurrentListStatusFilter() -> String {
+    class func keyForCurrentListStatusFilter() -> String {
         fatalError("You should implement this method in the subclass")
     }
 
-    func currentFilterIndex() -> Int {
+    class func currentFilterIndex() -> Int {
```

Comment:
I moved these to class methods since they need to be accessed elsewhere for preloading. I didn't want to make an instance of `PostListViewController` just to access the current filter settings. 

I agree that having a bunch of class methods on a VC is ugly, and definitely agree that the setting should correspond. I'd really like to move them out of the VC entirely, as they should be in business logic somewhere, either a model or service. Thoughts on moving them to `PostService` instead?